connectors/asset/README.md

```diff
@@ -0,0 +1,127 @@
+|                   |name                   |character field|
+|                   |type                   |character field|
+|                   |initial_value           |float field|
+|                   |site                    |foreign key relationship with Site|
+|                   |vulnerabilities        |many-to-many relationship with Vulnerability|
+|ip_addresses          |pk                     |primary key    |
+|                   |name                   |character field|
+|                   |address                   |character field|
+|                   |asset           |A foreign key relationship with Asset|
+|mac_addresses         |pk                     |primary key    |
+|                   |address                   |character field|
+|                   |asset           |A foreign key relationship with Asset|
+|                   |ip_addresses                   |A many-to-many relationship with the IPAddress model.|
+|hosts               |pk                     |primary key    |
+|                   |host                   |character field|
+|                   |asset           |A foreign key relationship with Asset|
+|ports         |pk                     |primary key    |
+|                   |port_number                   |integer field|
+|                   |layer7application                   |character field|
+|                   |protocol                |character field|
+|                   |apps           |A many-to-many relationship with App|
+|                   |ip_addresses                   |A many-to-many relationship with the IPAddress model.|
+|apps         |pk                     |primary key    |
+|                   |name                   |character field|
+|                   |assets           |A many-to-many relationship with Asset|
+
+![Asset Schema](docs/assetModel.png)
+For a comprehensive understanding of the schema, refer: ```https://github.com/IBM/cp4s-car-connectors/blob/develop/connectors/reference_connector/server/assets/models.py```
+
+III. INSTALLATION:
+-----------------------------------------------------------------
+- Requirements.txt file attached.
+
+
+IV. EXAMPLE USAGE (command line):
+-----------------------------------------------------------------
+
+```Usage: app.py [arguments]```
+
+required arguments in command line:
+
+```
+positional arguments:
+
+-host                       :base URL / root URL(https://raw.githubusercontent.com/IBM/cp4s-car-connectors/develop/sample/data)
+-car-service-url            :CAR DB url
+-car-service-key            :api_key
+-car-service-password       :password
+-source                     :data source
+```
+
+## Connector
+
+Install python dependencies
+```
+pip3 install -r requirements.txt
+```
+
+Running the connector:
+```
+python3 app.py -car-service-url=http://localhost:3000/api/car/v2 -car-service-key=none -car-service-password=none -source=test -host=https://raw.githubusercontent.com/mukeshreddy44/cp4s-car-connectors/dummy_connector/data
```

Comment:
Can this be presented in a more general fashion? We shouldn't be referencing a personal github in the README.
For example, see how we do it for other connectors: 
https://github.com/IBM/cp4s-car-connectors/blob/develop/connectors/crowdstrike/README.md?plain=1#L183
https://github.com/IBM/cp4s-car-connectors/blob/develop/connectors/nozomi/README.md?plain=1#L203